examples/employee-app/README.md

```diff
@@ -0,0 +1,182 @@
+
+# Helidon Quickstart SE - Employee Directory Example
+
+This project implements an employee directory REST service using Helidon SE. The application is composed of a Helidon REST Microservice backend along with an HTML/JavaScript front end. The source for both application is included with the Maven project.
+
+By default the service uses a ArrayList backend with sample data. You can connect the backend application to an Oracle database by changing the values in the `resources/DbCreds.properties` file.
+
+## Prerequisites
+
+1. Maven 3.5 or newer
+2. Java SE 8 or newer
+3. Docker 17 or newer to build and run docker images
+4. Kubernetes minikube v0.24 or newer to deploy to Kubernetes (or access to a K8s 1.7.4 or newer cluster)
+5. Kubectl 1.7.4 or newer to deploy to Kubernetes
+
+Verify prerequisites
+
+```sh
+java -version
+mvn --version
+docker --version
+minikube version
+kubectl version --short
+```
+
+## Using Oracle DB
+
+This application supports two storage implementations one using an array and other using an Oracle database to persist the data. To compile and package the application you need the JDBC driver. The JDBC driver for the Oracle DB is only available from the
+https://www.oracle.com/webfolder/application/maven/index.html[Oracle Maven Repository].
+
+This means that you have to configure the repository in order to add the driver
+ to your classpath.
+
+Follow these steps:
+
+- Go to https://www.oracle.com/webfolder/application/maven/index.html and
+ accept the license
+- Create an OTN account
+- Update your settings.xml as documented
+ https://docs.oracle.com/middleware/1213/core/MAVEN/config_maven_repo.htm#MAVEN9016[here]
+ 
+
+## Build
+
+```sh
+mvn package
+```
+
+## Start the application
+
+```sh
+cd target
+java -jar employee-app.jar
+```
+
+**Note:** For the static elements of the application to work, you must start the application from the target directory.
+
+
+## Exercise the application
+Get all employees.
+```json
+curl -X GET curl -X GET http://localhost:8080/employees
+```
+Output:  
+&lbrack;`{"birthDate":"1970-11-28T08:28:48.078Z","department":"Mobility","email":"Hugh.Jast@example.com","firstName":"Hugh","id":"100","lastName":"Jast","phone":"730-715-4446","title":"National Data Strategist"}`  

```

Comment:
This does not render well, why the html encoded character?

The curl command should probably be using \`\`\`bash and the json snippet \`\`\`json

Reply:
This should be fixed now. Redid all the examples.